python/thunder/rdds/fileio/imagesloader.py

```diff
@@ -133,6 +133,98 @@ def toArray(idxAndBuf):
         newDims = tuple(list(dims[:-1]) + [nplanes]) if nplanes else dims
         return Images(readerRdd.flatMap(toArray), nrecords=nrecords, dims=newDims, dtype=dtype)
 
+    def fromOCP (self, dataPath, resolution, serverName='ocp.me', startIdx=None, stopIdx=None, minBound=None, maxBound=None ):
```

Comment:
There are quite a few deviations from PEP conventions for spacing throughout here, can you please open this in PyCharm and try to fix them? That will be more efficient than my marking all of them.